libs/common-services/src/main/java/uk/gov/di/ipv/core/library/domain/VerifiableCredential.java

```diff
@@ -33,6 +38,14 @@ private VerifiableCredential(String userId, String criId, SignedJWT signedJwt)
             throw new CredentialParseException(
                     "Failed to get jwt claims to construct verifiable credential", e);
         }
+
+        try {
+            VerifiableCredentialParser.parseCredential(claimsSet);
+        } catch (CredentialParseException e) {
+            // For now, we just log a warning here that we can fix
+            // In future this should return a CredentialParseException instead
+            LOGGER.warn(LogHelper.buildErrorMessage("Failed to parse verifiable credential", e));
```

Comment:
This doesn't seem to be generating much that leads to "why" it failed to parse. I've run the acceptance tests against this to see what it produces, and the error messages look like:
```
{
...
  "message": {
    "description": "Failed to parse verifiable credential",
    "errorDescription": "uk.gov.di.ipv.core.library.exceptions.CredentialParseException: Failed parse verifiable credential"
  },
...
```

Reply:
Hmm, I'd have expected the underlying exception to form part of the stack trace. Is that not being included?

Reply:
Not in the logs I'm streaming from my deployed stack

Reply:
Ok, I'll see if we can get more detail - not much use if we lose the original message

Reply:
This is straight from CloudWatch
```
{
    "timestamp": "2024-04-29T09:53:53.831+0000UTC",
    "instant": {
        "epochSecond": 1714384433,
        "nanoOfSecond": 831076563
    },
    "thread": "main",
    "level": "WARN",
    "loggerName": "uk.gov.di.ipv.core.library.domain.VerifiableCredential",
    "message": {
        "description": "Failed to parse verifiable credential",
        "errorDescription": "uk.gov.di.ipv.core.library.exceptions.CredentialParseException: Failed parse verifiable credential"
    },
    "endOfBatch": false,
    "loggerFqcn": "org.apache.logging.log4j.spi.AbstractLogger",
    "threadId": 1,
    "threadPriority": 5,
    "clientId": "orchestrator",
    "coldStart": "false",
    "componentId": "https://dev-chrisw.01.dev.identity.account.gov.uk",
    "functionArn": "arn:aws:lambda:eu-west-2:130355686670:function:build-user-identity-dev-chrisw:live",
    "functionMemorySize": "1024",
    "functionName": "build-user-identity-dev-chrisw",
    "functionVersion": "268",
    "function_request_id": "d96b1fe8-7d03-4131-bbe7-3a5459cd0988",
    "govuk_signin_journey_id": "09a3e119-e9df-4221-8f80-28a5f630a9ff",
    "ipvSessionId": "i9s94p1uD391OLgQASwey5rEBcVCG-xOG6c8F1EKnkw",
    "samplingRate": "0.0",
    "service": "build-user-identity-dev-chrisw",
    "xray_trace_id": "1-662f6e31-57a91daa2b6136a64a41304a"
}
```